.travis.yml

```diff
@@ -3,7 +3,7 @@ rust:
   - stable
   - beta
   - nightly
-  - 1.21.0 # first cargo that supports [patch]
+  - 1.31.0
```

Comment:
Why so high? I think we can be compatible with lower versions.

Reply:
That's what I got when looking up when const_fn was stabilized (that was error on travis). I might be wrong though.